### PR TITLE
chore(bump deps) ember 2.16

### PR DESCRIPTION
**Pull request by kybishop**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -9,12 +9,18 @@ module.exports = {
...
-    browser: true
+    browser: true,
+    es6: true
   },
   rules: {
     'arrow-parens': ['error', 'as-needed'],
     // Destructuring is not supported in node 4.8.0
     'ember-suave/prefer-destructuring': 'off',
-    'operator-linebreak': ['error', 'before', { 'overrides': { '=': 'ignore'} } ]
+    'operator-linebreak': ['error', 'before', { 'overrides': { '=': 'ignore'} } ],
```
Comment (rwwagner90) [inline .eslintrc.js]: Do we need these custom rules here?

Comment (kybishop) [reply .eslintrc.js]: Need is a strong word. New versions of eslint prefer a strange indentation style that I believe is objectively less readable. The rules introduced here allow the current styling to pass linting checks without change.

Comment (rwwagner90) [reply .eslintrc.js]: So something in the eslint:recommended set changed?

Comment (kybishop) [reply .eslintrc.js]: Exactly. Indentation for function args was borked.

**Pull request merged**